src/k8s/pkg/k8sd/features/cilium/internal.go

```diff
@@ -0,0 +1,34 @@
+package cilium
+
+import (
+	"github.com/canonical/k8s/pkg/k8sd/types"
+)
+
+const (
+	// List of devices facing cluster/external network (used for BPF NodePort, BPF masquerading and host firewall); supports '+' as wildcard in device name, e.g. 'eth+'
```

Comment:
Should we just directly support regular expressions? IMHO just supporting `+` is a bit weird.

Reply:
This is not something we control, it is [upstream](https://docs.cilium.io/en/latest/network/kubernetes/kubeproxy-free/#nodeport-devices-port-and-bind-settings) implementation. They are only mentioning the `+` option and not explicitly saying it's regex.

Reply:
Right, I noticed this later in the review but forgot to remove my comment. Thanks